===================================================================
RCS file: /project/rfc2388/cvsroot/rfc2388/rfc2388.lisp,v
retrieving revision 1.9
retrieving revision 1.10
diff -u -r1.9 -r1.10
--- rfc2388/rfc2388.lisp	2005/08/02 09:01:47	1.9
+++ rfc2388/rfc2388.lisp	2006/07/27 07:40:46	1.10
@@ -439,8 +439,9 @@
                                                      :direction :output
                                                      ;; external format for faithful I/O
                                                      ;; see <http://cl-cookbook.sourceforge.net/io.html#faith>
-                                                     #+(or :lispworks :allegro)
+                                                     #+(or :sbcl :lispworks :allegro)
                                                      :external-format
+                                                     #+sbcl :latin-1
                                                      #+:lispworks '(:latin-1 :eol-style :lf)
                                                      #+:allegro (excl:crlf-base-ef :latin1))
                              (read-until-next-boundary input boundary nil out-file))
